source/Models/Event.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace source.Models
+{
+    public class Event
+
+    {
+        // unique string for event identifier
+        public int eventId { get; set; }
+
+        // reference to the organizer who created the event.
+        public string organizerId { get; set; }
+
+        //reference to the identifier of the guest list
+        public int guestListId { get; set; }
```

Comment:
I'm not sure what you mean here.  Do you mean that the guestListId is handled by the db?  That's correct, but I have the setter/getter in there so that the value can be captured.  Is there a better way to handle this?